<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWidget</class>
 <widget class="QWidget" name="mainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(82, 158, 218);

QWidget {
  background-color: #fff;
  border-radius: 12px;
}

QLabel {
  color: #464d55;
  font-weight: 600;
}
QLabel#heading {
  color: #0f1925;
  font-size: 18px;
  margin-bottom: 10px;
}

QLabel#subheading {
  color: #0f1925;
  font-size: 12px;
  font-weight: normal;
  margin-bottom: 10px;
}
QLineEdit {
  color: #464d55;
  font-weight: 600;
  border-radius: 8px;
  border: 1px solid #e0e4e7;
  padding: 5px 15px;
}

QLineEdit:focus {
  border: 1px solid #d0e3ff;
}

QLineEdit::placeholder {
  color: #767e89;
}
QPushButton {
  background-color: rgb(82, 158, 218);
  color: #fff;
  font-weight: 600;
  border-radius: 8px;
  border: 1px solid rgb(82, 158, 218);
  padding: 5px 15px;
  margin-top: 10px;
  outline: 0px;
}
QPushButton:hover {
  border: 3px solid #9ac3fe;
}
QPushButton:focus {
  border: 3px solid #9ac3fe;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="controllerWidget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget {
  background-color: #fff;
  border-radius: 12px;
}
QLabel {
  color: #464d55;
  font-weight: 600;
}
QLabel#heading {
  color: #0f1925;
  font-size: 18px;
  margin-bottom: 10px;
}

QLabel#subheading {
  color: #0f1925;
  font-size: 12px;
  font-weight: normal;
  margin-bottom: 10px;
}
QLineEdit {
  color: #464d55;
  font-weight: 600;
  border-radius: 8px;
  border: 1px solid #e0e4e7;
  padding: 5px 15px;
}

QLineEdit:focus {
  border: 1px solid #d0e3ff;
}

QLineEdit::placeholder {
  color: #767e89;
}
QPushButton {
  background-color: rgb(82, 158, 218);
  color: #fff;
  font-weight: 600;
  border-radius: 8px;
  border: 1px solid rgb(82, 158, 218);
  padding: 15px 30px;
  margin-top: 10px;
  outline: 0px;
}
QPushButton:hover {
  border: 3px solid #9ac3fe;
}
QPushButton:focus {
  border: 3px solid #9ac3fe;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QPushButton" name="currentTripButton">
        <property name="text">
         <string>Current Trip</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="futureTripButton">
        <property name="text">
         <string>Future Trips</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pastTripButton">
        <property name="text">
         <string>Past Trips</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="accountWidget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="acountDetailsLabel">
           <property name="text">
            <string>accountDetails</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="tripListWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget {
  background-color: #fff;
  border-radius: 12px;
}
QLabel {
  color: #464d55;
  font-weight: 600;
}
QLabel#tripListHeading {
  color: #0f1925;
  font-size: 30px;
  margin-bottom: 10px;
}
QLineEdit {
  color: #464d55;
  font-weight: 600;
  border-radius: 8px;
  border: 1px solid #e0e4e7;
  padding: 5px 15px;
}

QLineEdit:focus {
  border: 1px solid #d0e3ff;
}

QLineEdit::placeholder {
  color: #767e89;
}
QPushButton {
  background-color: rgb(82, 158, 218);
  color: #fff;
  font-weight: 600;
  border-radius: 8px;
  border: 1px solid rgb(82, 158, 218);
  padding: 5px 15px;
  margin-top: 10px;
  outline: 0px;
}
QPushButton:hover {
  border: 3px solid #9ac3fe;
}
QPushButton:focus {
  border: 3px solid #9ac3fe;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="tripListScrollWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>432</width>
           <height>262</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="tripListHeading">
            <property name="text">
             <string>titleList</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
